Assets/Scripts/Enemy/Enemy.cs

```diff
@@ -16,8 +16,7 @@ public int Cost
     {
         get { return CollisionDamage * (int) speed; }
     }
-
-    public HealthBar.Color EnemyType { get; protected set;}
+    public HealthBarColor enemyType { get; protected set; }
```

Comment:
Les propriétés doivent être déclarées avec une majuscule au début.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-properties